src/amperity/gocd/secret/vault/plugin.clj

```diff
@@ -42,16 +42,16 @@
 
 (defmulti handle-request
   "Handle a plugin API request and respond. Methods should return a map containing the following 3 keys:
-          ```{
-            :response-code     <int: the returned status, follows HTTP status conventions>
-            :response-body     <json-coercible: the response body, will be coerced into JSON>
-            :response-headers  <map: the response headers, follows HTTP header conventions>
-          }```
-
-          Params:
-          - `vault-client`: vault.client, used for auth and retrieval of all the secret values
-          - `req-name`: string, determines how to dispatch among implementing methods, essentially the route
-          - `data`: map, the body of the message passed from the GoCD server"
+  ```{
```

Comment:
the opening and closing parens should (still) be on the same line as the body:

```{:response-code
      :response-headers ... }```

Reply:
@pyncc I fixed that in 682d18b

I think GitHub was just playing tricks with its code diff updates